.github/workflows/build-images.yml

```diff
@@ -10,6 +10,9 @@ on:  # yamllint disable-line rule:truthy
       - main
     paths:
       - 'services/**'
+  pull_request:
```

Comment:
So this only triggers on a pull request merge, or what is a `pull_requests`?

Reply:
This configures the action to run on any pull request where there are changes to the `services/**` path. 

For example, a change to the README will not trigger this action.